google-cloud-bigquery/acceptance/bigquery/table_test.rb

```diff
@@ -401,6 +401,32 @@
     end
   end
 
+  it "updates its schema, loading from a File" do
+    begin
+      t = dataset.create_table "table_schema_test"
+      t.schema do |s|
+        s.integer "id", description: "id description", mode: :required
+        s.string "breed", description: "breed description", mode: :required
+        s.string "name", description: "name description", mode: :required
+        s.record "features", description: "features description", mode: :repeated do |s2|
+          s2.string "feature", description: "feature description", mode: :required
+        end
+      end
+      t.headers.must_equal %i[id breed name features]
+
+      t.schema replace: true do |s|
```

Comment:
I don't think it will matter whether `replace` is `true` or `false` here, will it? It would be good to update the `Table#schema` docs to cover loading--with an example, and an explanation of how it will always be a complete replacement.

Reply:
Good questions! Can you add tests/assertions to see what happens, then document in `Table#schema` docs?